Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -252,6 +251,17 @@ extension NSManagedObjectContext {
         
         delete(dto)
     }
+
+    func cleanChannels(cids: Set<ChannelId>) {
+        for cid in cids {
+            guard let channelDTO = channel(cid: cid) else { continue }
```

Comment:
Would it make sense to have a simple fetch request to load all channels in batch other than fetching each one separately?

Reply:
Done!